types/clockpicker/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "dt-header": false,
```

Comment:
`dt-header` rule shouldn't be disabled, 
usually there should be only `{"extends": "dtslint/dt.json"}` in this file

Reply:
ClockPicker is only at version 0.0.7, therefore if I don't set this rule to false I would have to write 0.0 which I found weird in the index.d
ts file.

What are your thoughts on that matter?

Reply:
I believe `0.0` is OK https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/isomorphic-fetch/index.d.ts#L1
`dt-header` rule is not just lint the version

Reply:
Alright I made the modification